graphix/command.py

```diff
@@ -51,10 +51,19 @@ class N(_KindChecker):
 
 
 @dataclasses.dataclass
-class M(_KindChecker):
-    """Measurement command. By default the plane is set to 'XY', the angle to 0, empty domains and identity vop."""
+class BaseM:
+    """Base class for measurement command.
+
+    This class does not contain any data except for the node to measure.
+    """
```

Comment:
The [documentation](https://github.com/TeamGraphix/graphix/commit/ad182700f4788771e8ff0a8581ba4468bca1e518#diff-30c3360a99df7f4c7b7d4e732b003eb6187cdec903d6b4a34d5c686b3b2d55f0R50-R60) added in commit ad182700 and later deleted in #205 was more informative. I believe we should revert to that version.

```suggestion
class BaseM:
    """Base measurement command.

    Represent a measurement of a node. In MBQC, a measure is an instance of `M`,
    with given plane, angles, and domains. In the context of blind computations,
    the server only knows which node is measured, and the parameters are given
    by the :class:`graphix.simulator.MeasureMethod` provided by the client.
    """

    """
```

Reply:
I intentionally trimmed them to ensure that no outdated info. is left.
Of course willing to revert to the original after discussion.